environment/templates/geth-deployment.yml

```diff
@@ -0,0 +1,154 @@
+objectmeta:
+  name: geth-dev-network
+spec:
+  template:
+    spec:
+      volumes:
+      - name: geth-configmap-volume
+        volumesource:
+          configmap:
+            localobjectreference:
+              name: gdmap
+      containers:
+      - name: geth-network
+        image: ethereum/client-go:v1.10.3
+        entrypoint: sh ./root/init.sh
+        volumemounts:
+        - name: geth-configmap-volume
+          mountpath: /root/config
+        - name: geth-configmap-volume
+          mountpath: /root/.ethereum/devchain/keystore/key1
+          subpath: key1
```

Comment:
Trying to figure out why mounting the whole directory with `readonly: false` doesn't work, geth still has wrong access to that dir, tried 0777/0644.